src/material-experimental/mdc-tabs/tab-nav-bar/tab-nav-bar.spec.ts

```diff
@@ -309,6 +315,123 @@ describe('MDC-based MatTabNavBar', () => {
     expect(instance.tabNavBar.selectedIndex).toBe(1);
   });
 
+  describe('without panel', () => {
+    let fixture: ComponentFixture<SimpleTabNavBarTestApp>;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(SimpleTabNavBarTestApp);
+      fixture.detectChanges();
+    });
+
+    it('should have no explicit roles', () => {
```

Comment:
nit: if I were writing these tests, I'd combine all the `with panel` and `without panel` attribute checks into a single test since they're basically checking a single use case.

Reply:
Acknowledged. I took a stab at refactoring these specs into a two specs (one for with panel and one for without) and they became very large and hard to work with. I'm going to keep them as-is.